pyproject.toml

```diff
@@ -17,20 +17,16 @@ packages = [
     { include = "home_assistant_bluetooth", from = "src" },
 ]
 
-[tool.poetry.build]
-generate-setup-file = true
-script = "build_ext.py"
-
 [tool.poetry.urls]
 "Bug Tracker" = "https://github.com/home-assistant-libs/home-assistant-bluetooth/issues"
 "Changelog" = "https://github.com/home-assistant-libs/home-assistant-bluetooth/blob/main/CHANGELOG.md"
 
 [tool.poetry.dependencies]
-python = "^3.9"
+python = ">=3.10,<3.13"
```

Comment:
Are there any strong reasons for forbidding 3.13 here? This is the only dependency of Home Assistant which disallows Python 3.13.